react/src/lib/components/DeckGLMap/layers/layersDefaultProps.ts

```diff
@@ -105,6 +105,10 @@ export const layersDefaultProps: Record<string, unknown> = {
         logCurves: true,
         refine: true,
         visible: true,
+        wellNameVisible: true,
```

Comment:
Not sure if this should be default visible currently, because it looks cluttered in the examples